server/tests/host_test.cpp

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include <vector>
+#include <numeric>
+#include <map>
+#include <string>
+
+#include "../host/host.cpp"
+
+#include "encryption/encrypt.h"
+#include "encryption/serialization.h"
+#include "utils.h"
+
+using namespace std;
+
+int main(int argc, char* argv[]) 
+{  
+    size_t accumulator_length = 3;
+    unsigned char*** encrypted_accumulator = new unsigned char**[accumulator_length * sizeof(unsigned char**)];
```

Comment:
`delete` all the arrays created using `new` to prevent memory leak (here and below)

Reply:
This seems to be pending

Reply:
Ah sorry I thought this was for the ecalls.cpp & assumed it was fine to leave dangling pointers in a main function